### PR TITLE
Create pork-intestine-blood.srt

### PR DESCRIPTION
**Pull request by immoonancient**
#494

### CODE REVIEW

File: subtitles/xue-yu/20191215-pork-intestine-blood.srt [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+
+6
+00:00:21,640 --> 00:00:23,000
+# 是不是听起来会觉得
+Perhaps it sounds like
+
+7
+00:00:23,000 --> 00:00:24,760
+# 有些疼痛呢
+a little bit pain?
```
Comment (NoraZheng) [inline subtitles/xue-yu/20191215-pork-intestine-blood.srt]:
```suggestion
a little bit painful?
```

File: subtitles/xue-yu/20191215-pork-intestine-blood.srt [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+
+22
+00:01:55,960 --> 00:01:57,660
+# 你就能够闻到
+instantly, you could smell
+
+23
+00:01:57,980 --> 00:01:59,620
+# 香芹的清香味道
+the refreshening aroma of parsley # 香芹 = parsley 加入对译表
```
Comment (NoraZheng) [inline subtitles/xue-yu/20191215-pork-intestine-blood.srt]:
```suggestion
the refreshing aroma of celery
```

File: subtitles/xue-yu/20191215-pork-intestine-blood.srt [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+
+29
+00:02:12,560 --> 00:02:14,840
+# 有些许的油花
+with a little bit of oil floating on top
+
+30
+00:02:20,640 --> 00:02:23,600
+# 汤 喝上去 很清新
+The soup tastes very refreshening
```
Comment (NoraZheng) [inline subtitles/xue-yu/20191215-pork-intestine-blood.srt]:
```suggestion
The soup tastes very refreshing
```

File: subtitles/xue-yu/20191215-pork-intestine-blood.srt [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+
+32
+00:02:33,640 --> 00:02:35,640
+# 好辣啊
+So spicy!
+
+33
+00:02:35,640 --> 00:02:38,400
+# 是胡椒粉的辣味
+It’s the spiciness of pepper power
```
Comment (NoraZheng) [inline subtitles/xue-yu/20191215-pork-intestine-blood.srt]:
```suggestion
It’s the spiciness of ground pepper
```

File: subtitles/xue-yu/20191215-pork-intestine-blood.srt [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+
+30
+00:02:20,640 --> 00:02:23,600
+# 汤 喝上去 很清新
+The soup tastes very refreshening
+
+31
+00:02:23,600 --> 00:02:28,000
+# 还有胡椒粉辛辣的味道
+and also spicy for the pepper power
```
Comment (NoraZheng) [inline subtitles/xue-yu/20191215-pork-intestine-blood.srt]:
```suggestion
and also spicy for the ground pepper
```

**Comment by NoraZheng** [comment]
国内非西餐极少见到parsley

File: docs/translation-table.md [hunk 6]
```diff
@@ -50,6 +50,7 @@
 |     洋葱     |            onion             |
 |     西芹     |            celery            |
 |     芹菜     |         leaf celery          |
+|     香芹     |            parsley           |
```
Comment (immoonancient) [inline docs/translation-table.md]:
```suggestion
```

**Pull request merged**